msm/src/serialization/witness.rs

```diff
@@ -138,14 +138,18 @@ pub fn deserialize_field_element<Fp: Field, Env: InterpreterEnv<Fp>>(
     assert!(limbs[1] < 2u128.pow(88));
     assert!(limbs[2] < 2u128.pow(79));
 
-    env.copy(&input_limb0, kimchi_limbs0);
-    env.copy(&input_limb1, kimchi_limbs1);
+    let limb0_var = env.copy(&input_limb0, kimchi_limbs0);
+    let limb1_var = env.copy(&input_limb1, kimchi_limbs1);
     let limb2_var = env.copy(&input_limb2, kimchi_limbs2);
 
+    let limb2_0_var = {
```

Comment:
Why this change? It looks equivalent to me at first sight

Reply:
It is equivalent. It was simply to use limb2_0_var below. Certainly a useless change. Let me remove it.

Reply:
If it is fine for you, I will remove it after https://github.com/o1-labs/proof-systems/pull/1892 is merged. Otherwise, it will create a conflict.

Reply:
Lemme know if it is better for you https://github.com/o1-labs/proof-systems/pull/1901